pkg/archive/archive.go

```diff
@@ -34,6 +36,7 @@ type packager struct {
 	manifest    map[string]struct{}
 	blobs       map[string]struct{}
 	packedBlobs map[string]struct{}
+	ctx         context.Context
```

Comment:
Typically you'd pass the context into the function itself so the caller can configure each call differently, ex. different timeouts.